src/api/axiosGeoJSON.ts

```diff
@@ -17,19 +17,27 @@ const instance = axios.create({
 const CancelToken = axios.CancelToken
 const source = CancelToken.source()
 
-const getGeoJsonData = async (type: string, UUID: string) => {
+const getGeoJsonData = async (
+    type: string,
+    UUID: string,
+    signal?: AbortSignal
+) => {
     const res = await instance.get(
         `ows?service=wfs&version=${api_version}&request=GetFeature&typeNames=OMGEVINGSBELEID:${type}&cql_filter=UUID=%27${UUID}%27&outputFormat=application/json`,
-        { cancelToken: source.token }
+        { cancelToken: source.token, ...(signal && { signal }) }
```

Comment:
I think we can remove the cancelToken since it's deprecated and signal will do the same thing.

Reply:
Or a more clean way:

```
const getGeoJsonData = async (
    type: string,
    UUID: string,
    config?: AxiosRequestConfig
) => {
const res = await instance.get(
        `ows?service=wfs&version=${api_version}&request=GetFeature&typeNames=OMGEVINGSBELEID:${type}&cql_filter=UUID=%27${UUID}%27&outputFormat=application/json`,
       { ...config }
```